__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const app = require("../db/app");
+const request = require("supertest");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return connection.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/topics", () => {
+  test("200: responds with topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        topics.forEach((topic) => {
```

Comment:
Great work using a forEach! As we're using this though, could you just put a line before this that checks the length of the array we're getting back please?